assets/src/edit-story/components/checklist/checklistContent/priorityChecks.js

```diff
@@ -73,13 +75,13 @@ export function PriorityChecks({
         <StoryMissingTitle />
         <StoryTitleLength />
         <StoryMissingExcerpt />
-        <StoryPosterAttached />
-        {/* TODO: #8129 this overlaps alot with aspect ratio, do we need both? */}
-        <StoryPosterPortraitSize />
-        <StoryPosterAspectRatio />
-        <PublisherLogoSize />
-        <VideoElementMissingPoster />
-        {isTranscodingEnabled && <VideoOptimization />}
+        {hasUploadMediaAction && <StoryPosterAttached />}
+        {/* TODO: #8129 this overlaps a lot with aspect ratio, do we need both? */}
+        {hasUploadMediaAction && <StoryPosterPortraitSize />}
+        {hasUploadMediaAction && <StoryPosterAspectRatio />}
+        {hasUploadMediaAction && <PublisherLogoSize />}
+        {hasUploadMediaAction && <VideoElementMissingPoster />}
+        {isTranscodingEnabled && hasUploadMediaAction && <VideoOptimization />}
```

Comment:
`isTranscodingEnabled` itself already checks for `hasUploadMediaAction`, so this is redundant.

Could remove this redundant check and add a comment explaining why.